### ISSUE TITLE
Has anyone successfully trained the fish suite provided by DM_Control?

### ISSUE THREAD

**Issue by benwang0912** [keras-rl/keras-rl#245]
I have been trying on this for a few days, but it seems that the fish has learned nothing.
I use [https://github.com/deepmind/dm_control](url) and [https://github.com/martinseilair/dm_control2gym](url) and DDPG network provided by keras-rl to do my work.
Is there any suggestion? Anything would be appreciated(even for suggestions for my poor grammar. I am not a native English speaker. Apologize for my poor grammar).
Thanks in advance!

**Comment by mirraaj** [comment]
@benwang0912 , Could you let me know the name of the equivalent gym environment, if there exits any?

**Comment by mirraaj** [comment]
What do you think could be the possible reason for this issue? 
Have you tried reducing the magnitude of the learning rate for the optimizer?

**Comment by benwang0912** [comment]
I guess it's because of the randomness of every episode.
The replay memory is full of dummy actions.
So the fish learned nothing from the replay memory.
It's all my guess, I don't have anything to proof that it is actually the reason.

I managed to solve a simpler environment by reducing the 3D environment into 2D.
And fixed the position of fish at (0, -0.2) and the target at (0.2, 0.3) every episode.
After the environment described above is solved, I did transfer learning on my trained network. 
I gave the position of target a little randomness, which means the target should always be initialed at the front-left or front-right of the fish.
And it works perfectly now.

Now, I am going to try adding more fish in my environment and try to avoid them from collision.

P.s.
I am using learning rate equals 1e-4 for both actor and critic network.
Is it small enough?

**Comment by mirraaj** [comment]
@benwang0912 were you trying to learn from a 3D observation space before?

You can assign the value of nb_max_start_steps = 0 while calling the agent. This will save you from random steps taken every episode. Please try this and let me know the results.

I think the learning rate is good for actor-critic learning agents. However, you should try 1e-5 and 5e-5 to compare your results.

**Comment by benwang0912** [comment]
Yes, I was trying to learn from 3D observation space.
Is it what cause the problem?

I am going abroad next week.
So, the testing may have to wait for a while.
I will let you know as soon as I tried those parameters.

Thanks again.

**Comment by mirraaj** [comment]
I have not tried 3D action space games yet! However, it should not be much of a problem. You might have to train the environment for a larger time to learn a geed set of parameters for the network.

Please let me know if you have any comments about the issue! Have a safe trip!
